rtf_vis_tool/src/Components/frustum_classes/view_frustum.js

```diff
@@ -0,0 +1,168 @@
+/*Contains functions to transform camera instrinsic variables into a frustum consisting of 5 vertices.
+
+Author: Adi Singh
+*/
+
+// Third-Party Package Imports.
+var nj = require('numjs');
+
+class ViewFrustum {
+    // Generates vertices of a 5-face mesh for drawing a pinhole camera in 3d.
+
+    constructor(fx, img_w, img_h, ray_len) {
+        /*Initializes the ViewFrustum Object.
+
+            Args:
+                fx (float): Focal length in x-direction.
+                img_w (float): Image width (in pixels).
+                img_h (float): Image height (in pixels).
+                ray_len (float): Extent to which frustum rays extend away from optical center.
+        */
+
+        this.fx_ = fx;
+        this.img_w_ = img_w;
+        this.img_h_ = img_h;
+        this.frustum_ray_len_ = ray_len;
+    }
+
+    normalize_ray_dirs(ray_dirs) {
```

Comment:
do we have a unit test on this fn?

Reply:
yup. It's in `ViewFrustum.test.js` on line 15